build-scripts/runtime_lib.cmake

```diff
@@ -1,6 +1,46 @@
 # Copyright (C) 2019 Intel Corporation. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
+file(WRITE "${PROJECT_BINARY_DIR}/try_compile_clock_nanosleep.c"
+"   #include <time.h>
+    void mysleep_ms(int milisec)
+    {
+        struct timespec res;
+        res.tv_sec = milisec/1000;
+        res.tv_nsec = (milisec*1000000) % 1000000000;
+        clock_nanosleep(CLOCK_MONOTONIC, 0, &res, NULL);
```

Comment:
should add error handling ?

Reply:
I don't think this code is intended to run, just compile.